src/ghci/parse/ghc_message/compilation_summary.rs

```diff
@@ -51,13 +101,10 @@ pub fn compilation_summary(input: &mut &str) -> PResult<CompilationSummary> {
     .parse_next(input)?;
     let _ = " module".parse_next(input)?;
     let _ = opt("s").parse_next(input)?;
-    let _ = " loaded.".parse_next(input)?;
-    let _ = line_ending_or_eof.parse_next(input)?;
+    let _ = ' '.parse_next(input)?;
+    let _ = alt(("loaded", "reloaded", "added", "unadded", "checked")).parse_next(input)?;
```

Comment:
GHC 9.10 says `Ok, 10 modules reloaded` instead of `Ok, 10 modules loaded`. There's some other special cases I've accounted for here, but that was the big one causing problems.